browser/ui/views/tabs/vertical_tab_strip_browsertest.cc

```diff
@@ -233,20 +233,35 @@ IN_PROC_BROWSER_TEST_F(VerticalTabStripBrowserTest, WindowTitle) {
   browser()->profile()->GetPrefs()->SetBoolean(prefs::kUseCustomChromeFrame,
                                                true);
 #endif
+
   // Pre-condition: Window title is "hidden" by default on vertical tabs
```

Comment:
nit: it seems this comment needs to be updated.

Reply:
Yeah, that's right. Updated.